substrate/frame/support/src/traits/voting.rs

```diff
@@ -25,14 +25,21 @@ use sp_runtime::{traits::Member, DispatchError};
 use sp_std::prelude::*;
 
 pub trait VoteTally<Votes, Class> {
+	/// Initializes a new tally
 	fn new(_: Class) -> Self;
+	/// Returns the number of positive votes for the tally.
 	fn ayes(&self, class: Class) -> Votes;
+	/// Returns the approval ratio (positive to total votes) for the tally, without multipliers (i.e. conviction, ranks, etc.).
```

Comment:
Line length + "i.e." -> "e.g."
```suggestion
	/// Returns the approval ratio (positive to total votes) for the tally, without multipliers
	/// (e.g. conviction, ranks, etc.).
```